### PR TITLE
feat: update ignite pkg

### PR DESCRIPTION
**Pull request by mazzy89**

### CODE REVIEW

**Comment by mazzy89** [comment]
```go
	if legacySendCmd {
		ccoptions = append(ccoptions, chaincmd.WithLegacySendCommand())
	}

	switch sdkVersion {
	case "stargate-44":
		ccoptions = append(ccoptions,
			chaincmd.WithVersion(cosmosver.StargateFortyFourVersion),
		)
	case "stargate-40":
		ccoptions = append(ccoptions,
			chaincmd.WithVersion(cosmosver.StargateFortyVersion),
		)
	case "launchpad":
		ccoptions = append(ccoptions,
			chaincmd.WithVersion(cosmosver.MaxLaunchpadVersion),
			chaincmd.WithLaunchpadCLI(appCli),
		)
		if home != "" {
			ccoptions = append(ccoptions, chaincmd.WithLaunchpadCLIHome(home))
		}
	default:
		ccoptions = append(ccoptions,
			chaincmd.WithVersion(cosmosver.Latest),
		)
	}
```

`WithLegacySendCommand` does not exist anymore and I guess we can remove it.

Also in the switch clause we can remove the other cases and leave only the default. What do you think @ilgooz?

**Comment by jeronimoalbi** [comment]
I believe that the `go.mod` replace line can be removed:

```
replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

**Review by jeronimoalbi — APPROVED** [review]

**Comment by mazzy89** [comment]
@jeronimoalbi I've removed it. However there are still other stuff to remove which I would need to have advice.

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,6 @@ module github.com/tendermint/faucet
 go 1.16
```
Comment (Pantani) [inline go.mod]:
```suggestion
go 1.21
```

**Comment by mazzy89** [comment]
@Pantani I have updated everything and removed the old stuff. Please @Pantani another look.

**Review by Pantani — APPROVED** [review]

**Comment by Pantani** [comment]
@mazzy89 I don't have rights to merge it

**Comment by mazzy89** [comment]
@Pantani thanks for the review. It is not clear to me whether we want to merge the changes to the upstream or leave them in the fork. @ilgooz can you advice here?

**Pull request closed without merging**